src/Symfony/Component/Polyfill/Tests/IntlTest.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Polyfill\Tests;
+
+use Symfony\Component\Polyfill\Intl as p;
+use Normalizer as n;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ *
+ * @covers Symfony\Component\Polyfill\Intl::<!public>
+ * @requires extension intl
+ */
+class IntlTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @covers Symfony\Component\Polyfill\Intl::grapheme_extract
+     */
+    public function testGrapheme_extract_arrayError()
+    {
+        try {
+            p::grapheme_extract(array(), 0);
+            $this->fail('Warning or notice expected');
+        } catch (\PHPUnit_Framework_Error_Warning $e) {
+            $this->assertTrue(true, 'Regular PHP throws a warning');
+        } catch (\PHPUnit_Framework_Error_Notice $e) {
+            $this->assertTrue(true, 'HHVM throws a notice');
+        }
```

Comment:
shouldn't it actually use a different expectation based on HHVM vs PHP ? There is nothing ensuring that the stub produces a notice on HHVM here